ui-tests/src/test/java/io/syndesis/qe/steps/connections/ConnectionSteps.java

```diff
@@ -45,17 +47,17 @@ public void verifyConnectionDetails(String connectionName) {
 
 	@Then("^Camilla can see \"(\\w+)\" connection$")
 	public void expectConnectionTitlePresent(String connectionName) {
-		listComponent.getConnectionByTitle(connectionName).shouldBe(visible);
+		listPage.getListComponent().getConnectionByTitle(connectionName).shouldBe(visible);
```

Comment:
I'm not sure but i don't see benefits of going trough the whole application structure to get desired page object. I think the original variant is sufficient.

Reply:
Imho, it's very similar just the reference to listComponent is nested in `ConnectionListPage`

Reply:
Yes, sure but, if you don't use ConnectionListPage in that step it seems unnecessary to me.

Reply:
The original code or the new one will instantiate those fields for every step even when it's not used. Lets ask the question what's the purpose of this changed actually?  I think, it's uniformity of accessing components objects.

Reply:
OK

Reply:
@dsimansk 👍